hosts.go

```diff
@@ -134,6 +136,26 @@ func (h *Host) IPAddresses() map[string]string {
 	return ipAddresses
 }
 
+// IPAddressesAll returns all ipaddresses
+//
+// Its result is a map of a slice of string. This key means interface name
+// and this value means ipaddresses bound to this interface.
+func (h *Host) IPAddressesAll() map[string][]string {
```

Comment:
It is good to have another implementation for all ipaddresses, and how about split it into two methods (one is for ipv4, and another is for ipv6) ?